tox.ini

```diff
@@ -1,7 +1,12 @@
 [tox]
+min_version = 4.4.4
 envlist = py37,py38,py39,py310,py311,pypy3,coverage,docs,quality
+labels =
+    ci-tests = py{37,38,39,310,311,py3}
```

Comment:
When the list of supported Python versions changes in `envlist`, above, the `ci-tests` label should also be updated.

This uses Tox's generative environment syntax.